### PR TITLE
Update targetref.md

### PR DESCRIPTION
**Pull request by GigaPrimatus**
The total order defines the priority of the policy.  The order currently shown is the reverse of the explanation provided.

<Explain your change!>

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by lahabana** [comment]
I think this makes sense @lobkovilya can you double check?

File: app/_src/policies/targetref.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ because a proxy can be targeted by multiple `targetRef`'s.
 
 We define a total order of policies:
```
Comment (michaelbeaumont) [inline app/_src/policies/targetref.md:284]:
```suggestion
We define a total order of policy priority:
```

File: app/_src/policies/targetref.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ because a proxy can be targeted by multiple `targetRef`'s.
...
 
-- Mesh > MeshSubset > MeshService > MeshServiceSubset (the more a `targetRef` is focused the higher priority it has)
+- MeshServiceSubset > MeshService > MeshSubset > Mesh (the more a `targetRef` is focused the higher priority it has)
```
Comment (michaelbeaumont) [inline app/_src/policies/targetref.md:286]:
```suggestion
- MeshServiceSubset > MeshService > MeshSubset > Mesh (the more a `targetRef` is focused the higher priority it has)
```

**Review by michaelbeaumont — APPROVED** [review]

**Comment by michaelbeaumont** [comment]
This page needs to be reworked now that we have `MeshGateway` but this PR improves it as is.

**Pull request merged**